### DISCUSSION TITLE
Unable to save shortcut

### DISCUSSION THREAD

**Discussion by Dhruv-k-2** [SpotX-Official/SpotX#5] [Errors]
i get an error when running the bat file

![image](https://user-images.githubusercontent.com/81463019/130318573-451c7448-531f-4e1c-9af0-ad8265bce75c.png)

**Comment by amd64fox** [comment]
Hello, so far I can not catch a similar error.
Could you tell me which version of Windows and PowerShell are using?

Please check if you have changed the default folders, which are indicated in the screenshot

**Reply by amd64fox** [reply]
To check the existence of these directories, do the following:
Press `Win+R`  and enter this `%USERPROFILE%\Desktop`  - your desktop should open, 
or
`%APPDATA%\Spotify`  - the Spotify folder should open

If an error appears when calling these folders, then you have changed the default folders

**Reply by amd64fox** [reply]
Fixed thanks to this #14
